### ISSUE TITLE
Automated cleanup does not work if ZN failed to start normally

### ISSUE THREAD

**Issue by unsystemizer** [HelloZeroNet/ZeroNet#611]
While testing with Tails (see #610) ZeroNet stops or gets stuck while attempting to make a connection.
I stop it with CTRL+C and the next time I start, it can't auto-cleanup corrupt data. Something is wrong with the cleanup code. 

A workaround (because it can't connect, I don't have any data except the auto-generated address) is to `rm -rf data`.

**Comment by unsystemizer** [comment]
Error log:
```
amnesia@amnesia:~/.ssh/zeronet/ZeroNet-master$ python zeronet.py --proxy 127.0.0.1:9050 --tor disable
- Starting ZeroNet...
[16:44:21] - OpenSSL loaded, version: 01000114F
[16:44:21] - Patching sockets to socks proxy: 127.0.0.1:9050
[16:44:21] - Version: 0.4.1 r1536, Python 2.7.9 (default, Aug 13 2016, 16:41:35) 
[GCC 4.9.2], Gevent: 1.0.1
[16:44:21] - Creating FileServer....
[16:44:21] - Creating UiServer....
[16:44:21] SiteManager Deleting orphan site from content.db: 1Name2NXVi1RDPDgf5617UoW7xA6YrhM9F
Traceback (most recent call last):
  File "zeronet.py", line 16, in main
    main.start()
  File "src/main.py", line 420, in start
    actions.call(config.action, action_kwargs)
  File "src/main.py", line 120, in call
    func(**kwargs)
  File "src/main.py", line 130, in main
    ui_server = UiServer()
  File "src/Ui/UiServer.py", line 61, in __init__
    self.sites = SiteManager.site_manager.list()
  File "src/Site/SiteManager.py", line 130, in list
    self.load()
  File "plugins/MergerSite/MergerSitePlugin.py", line 301, in load
    super(SiteManagerPlugin, self).load(*args, **kwags)
  File "plugins/Zeroname/SiteManagerPlugin.py", line 17, in load
    super(SiteManagerPlugin, self).load(*args, **kwargs)
  File "src/Site/SiteManager.py", line 53, in load
    ContentDb.getContentDb().deleteSite(row["address"])
  File "src/Content/ContentDb.py", line 65, in deleteSite
    site_id = self.site_ids[site_address]
KeyError: u'1Name2NXVi1RDPDgf5617UoW7xA6YrhM9F'
```

**Comment by HelloZeroNet** [comment]
It should be fixed in 0.5.0
